README.md

```diff
@@ -2,3 +2,136 @@
 
 This document explains our approach to working with Rubocop in dxw. We should
 in general:
+
+### Basics
+- use the latest version of rubocop
+- configure CI to run rubocop and fail if there are *any* violations
```

Comment:
It's default rubocop behaviour to fail on a single infraction — I'm not sure if this needs the extra emphasis?

Reply:
Rephrased in a separate commit (to be squashed in)